lib/config/definitions.js

```diff
@@ -131,6 +131,12 @@ const options = [
     description: 'Requested reviewers for Pull Requests',
     type: 'list',
   },
+  {
+    name: 'gitlabRemoveSourceBranch',
+    description: 'Remove source branch on merge',
+    type: 'boolean',
+    default: false,
```

Comment:
What's your opinion on making it default to true? It doesn't sound to me like anyone should object to this change in behaviour as they can until the option manually as well, right?